helm_deploy/digital-prison-services/values.yaml

```diff
@@ -52,21 +52,6 @@ generic-service:
       GOOGLE_ANALYTICS_GA4_ID: "GOOGLE_ANALYTICS_GA4_ID"
       GOOGLE_TAG_MANAGER_ID: "GOOGLE_TAG_MANAGER_ID"
       SESSION_COOKIE_SECRET: "SESSION_COOKIE_SECRET"
-      NOTIFY_API_KEY: "NOTIFY_API_KEY"
-      WANDSWORTH_OMU_EMAIL: "WANDSWORTH_OMU_EMAIL"
-      THAMESIDE_OMU_EMAIL: "THAMESIDE_OMU_EMAIL"
-      HEWELL_OMU_EMAIL: "HEWELL_OMU_EMAIL"
-      BERWYN_OMU_EMAIL: "BERWYN_OMU_EMAIL"
-      ELMLEY_OMU_EMAIL: "ELMLEY_OMU_EMAIL"
-      NOTTINGHAM_OMU_EMAIL: "NOTTINGHAM_OMU_EMAIL"
-      BULLINGDON_OMU_EMAIL: "BULLINGDON_OMU_EMAIL"
-      PETERBOROUGH_OMU_EMAIL: "PETERBOROUGH_OMU_EMAIL"
```

Comment:
These are also configured in the k8s secrets. Probably manually? But need a note on the PR to remove them?

Reply:
There is a note on the PR.  Did you mean the Jira ticket?